src/test/persistence/secondary/mongo-secondary-entity-manager-test.ts

```diff
@@ -11,16 +11,6 @@ export class MongoSecondaryEntityManagerTest implements Test {
 
   private _keyGenParams = { prefix: 'some-prefix' };
 
-  private _idsForTest: string[] | number[] = ['8', '9', '10', '11', '12', '13'];
-
-  private _usersForTest: UserTest[] = [
-    { id: '8', name: 'Alex', country: 'Spain' },
-    { id: '9', name: 'Andrew', country: 'Germany' },
-    { id: '10', name: 'Dario', country: 'Spain' },
-    { id: '11', name: 'Steve', country: 'Germany' },
-    { id: '12', name: 'Steve A', country: 'Germany' },
-    { id: '13', name: 'Micaelo', country: 'Spain' },
-  ];
```

Comment:
@Adrianmjim I've moved this (it was used in two tests). I'm not sure this is a good idea. What do you think about that? I can undo this.

I've moved this because it was used in two tests and one of the usages was only to use one of the users. We've got another attribute (_userCollection) that is used for the same purpose. Should we unify both attributes in a single one?

Reply:
That is trash code. After test refactor I forgot to remove them.